src/test/java/com/google/sps/NonPersistentUserRepositoryTest.java

```diff
@@ -0,0 +1,116 @@
+package com.google.sps;
+
+import com.google.sps.data.NonPersistentUserRepository;
+import com.google.sps.data.User;
+import java.util.ArrayList;
+import java.util.Collection;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** */
+@RunWith(JUnit4.class)
+public final class NonPersistentUserRepositoryTest {
+
+  private static final User USER_A = new User("User A");
+
+  @Test
+  public void addFakeMentorsTest() {
+    NonPersistentUserRepository testDataRepo = new NonPersistentUserRepository();
+    testDataRepo.addFakeMentors();
+    String expected = "Andre Jerry Julie ";
+    Assert.assertEquals(expected, testDataRepo.toString());
```

Comment:
Ah, so this is a case where indeed you need your collection of AllUsers to be iterated in a specific order (since you compare .toString() to a specific string which assumes alphabetical order). You probably shouldn't use HashSet for storing users. No action required here, just a remark.

Reply:
Noted